plugins/web-plugin/src/main/java/ru/sbtqa/tag/stepdefs/ru/WebSteps.java

```diff
@@ -3,9 +3,9 @@
 import cucumber.api.java.Before;
 import cucumber.api.java.ru.И;
 import ru.sbtqa.tag.pagefactory.exceptions.PageInitializationException;
-import ru.sbtqa.tag.stepdefs.WebGenericStepDefs;
+import ru.sbtqa.tag.stepdefs.WebGenericSteps;
 
-public class WebStepDefs extends WebGenericStepDefs {
+public class WebSteps extends WebGenericSteps {
 
     @Before(order = 1)
     public void initWeb() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add the "@Override" annotation above this method signature ([squid:S1161](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1161))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=70&issues=AWSwZVrtIUupByjQG-fD&open=AWSwZVrtIUupByjQG-fD)

[ISSUE_KEY]: AWSwZVrtIUupByjQG-fD